src/oid.rs

```diff
@@ -114,6 +114,11 @@ impl ObjectId {
         }
     }
 
+    /// Retrieves the timestamp (seconds since epoch) from an ObjectId.
+    pub fn timestamp(&self) -> u32 {
```

Comment:
I think if it would be better to return a `DateTime` here instead of a number of seconds. This would be in line with the other drivers (Go / Ruby / Node / Python). Java opts to have methods for both, but I think that would probably be unnecessary given that a user could easily [get the number of seconds from the `DateTime`](https://docs.rs/chrono/0.4.11/chrono/struct.DateTime.html#method.timestamp).

cc @saghm

Reply:
Interesting; the version of this method we had before just returned the number of seconds, but I wouldn't be opposed to returning a DateTime here

Reply:
fixed